frontend/public/models/yaml-templates.ts

```diff
@@ -1210,6 +1210,48 @@ spec:
             - "-c"
             - "for i in 9 8 7 6 5 4 3 2 1 ; do echo $i ; done"
           restartPolicy: Never
+`,
+  )
+  .setIn(
+    [referenceForModel(k8sModels.VolumeSnapshotModel), 'default'],
+    `
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshot
+metadata:
+  name: new-snapshot-test
+spec:
+  volumeSnapshotClassName: csi-hostpath-snapclass
+  source:
+    persistentVolumeClaimName: pvc-test
+`,
+  )
+  .setIn(
+    [referenceForModel(k8sModels.VolumeSnapshotClassModel), 'default'],
+    `
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+  name: csi-hostpath-snapclass
+driver: hostpath.csi.k8s.io #csi-hostpath
+deletionPolicy: Delete
+`,
+  )
+  .setIn(
+    [referenceForModel(k8sModels.VolumeSnapshotContentModel), 'default'],
+    `
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotContent
+metadata:
+  name: new-snapshot-content-test
```

Comment:
Should we have "test" in the name if this is going to be the real volume snapshot template? (Here are elsewhere in the samples.)

We try to make the examples just work if you click save with no changes, but I imagine that's not possible here.

Reply:
> Should we have "test" in the name if this is going to be the real volume snapshot template? (Here are elsewhere in the samples.)
Right, we can call it an example/test. I'll change it.
> We try to make the examples just work if you click save with no changes, but I imagine that's not possible here.
I agree, Right now volume snapshots are not adapted to all the storage providers as it's not GA'ed. When snapshot gets GA'ed we should get the default CSI hostpath driver out of the box. So for now just added an example of it as this is provided on k8s documentation as well.